src/qibolab/instruments/qblox/module.py

```diff
@@ -0,0 +1,42 @@
+"""Qblox Cluster QCM driver."""
+
+from qibolab.instruments.abstract import Instrument
+from qibolab.instruments.qblox.port import QbloxInputPort, QbloxOutputPort
+
+
+class ClusterModule(Instrument):
+    DEFAULT_SEQUENCERS_VALUES = {
+        "cont_mode_en_awg_path0": False,
+        "cont_mode_en_awg_path1": False,
+        "cont_mode_waveform_idx_awg_path0": 0,
+        "cont_mode_waveform_idx_awg_path1": 0,
+        "marker_ovr_en": True,  # Default after reboot = False
+        "marker_ovr_value": 15,  # Default after reboot = 0
+        "mixer_corr_gain_ratio": 1,
+        "mixer_corr_phase_offset_degree": 0,
+        "offset_awg_path0": 0,
+        "offset_awg_path1": 0,
+        "sync_en": False,  # Default after reboot = False
+        "upsample_rate_awg_path0": 0,
+        "upsample_rate_awg_path1": 0,
+    }
+
+    def __init__(self, name: str, address: str):
+        """This class defines common features shared by all Qblox modules (QCM-
+        BB, QCM-RF, QRM-RF).
+
+        It serves as a foundational class, unifying the behavior of the
+        three distinct modules. All module-specific classes are intended
+        to inherit from this base class.
+        """
+
+        super().__init__(name, address)
+        self.ports: dict = {}
+
+    def port(self, name: str, out: bool = True):
```

Comment:
While working on something else, I realized that in the abstract instrument and other drivers we call this `ports`:
https://github.com/qiboteam/qibolab/blob/e6441d1f7b3f97c1fbc350059a10d32f58b4f0f6/src/qibolab/instruments/abstract.py#L85

It may be useful to follow the same convention here, so that the platform creation looks more similar between different instruments.

Alternatively, we could rename everything to `port`, which maybe also sounds better in terms of usability, however this would likely be a bigger change.

Reply:
Ok thanks @stavros11 I will rename this in `ports`.